### PR TITLE
Release new version with separate stylesheets per font

### LINKED ISSUE #37: Deploy .scss files
The bundled .css file includes everything: Accidenz Commons, Roboto Condensed, Source Sans Pro and the Icons. That's a lot of KBs for a website to load, specially if a website needs only one or two of those things (eg: a website that needs only the icons, or the icons and Roboto Condensed font).

We should deploy the .scss files separately to allow users to import modules individually

### PR DESCRIPTION
**Pull request by dhruvkb**
## Fixes

Fixes #37 by @brenoferreira 

## Description
While @brenoferreira's own solution #39 is a viable fix to deploy raw SCSS files to npm, it might not really solve the problem of serving independent CSS files. This PR might.

## Technical details
SCSS files being compiled on the client would fail as the font assets would not be present there. Precompiled CSS files on a CDN can directly link to the font assets on the CDN itself.

## Checklist
 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the `master` branch of the repository.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Review by hugosolar — APPROVED** [review]

File: src/index.js [hunk 1]
```diff
@@ -1 +1,4 @@
-require('./styles/fonts.scss')
+require('./scripts/accidenz_commons')
```
Comment (brenoferreira) [inline src/index.js]: Based on conversations with the team, `accidenz_commons` isn't used for text anywhere at the moment. So I think we can remove this from the main bundle

Comment (brenoferreira) [reply src/index.js]: Everything else looks good 👍

**Review by brenoferreira — CHANGES_REQUESTED** [review]

File: src/index.js (hunk 1 above)
Comment (dhruvkb) [reply src/index.js]: Resolved in 2ea0024.

**Review by brenoferreira — APPROVED** [review]

**Comment by dhruvkb** [comment]
Waiting on #45 before releasing a new version.

**Pull request merged**